src/Home.js

```diff
@@ -47,39 +24,11 @@ export default function Home() {
     setTodoDescription("");
   }
 
-  async function deleteTodo(todo) {
-    try {
-      await DataStore.delete(todo);
-    } catch (e) {
-      console.log(`Delete failed: ${e}`);
-    }
-  }
-
   function closeModal() {
     setModalVisible(false);
   }
 
-  const todoItem = ({ item }) => (
-    <Pressable style={styles.todoContainer}
-      onLongPress={() => {
-        deleteTodo(item);
-      }}>
-      <View>
-        <Text style={styles.todoHeading}>{item.name}</Text>
-        <Text style={styles.todoDescription}>{item.description}</Text>
-      </View>
-      <Pressable
-        style={[styles.checkbox, item.isComplete && styles.selectedCheckbox]}
-        onPress={() => {
-          setComplete(!item.isComplete, item)
-        }}
-      >
-        <Text style={[styles.checkboxText, item.isComplete && styles.selectedCheckboxText]}>{(item.isComplete) ? "✓" : ""}</Text>
-      </Pressable>
-    </Pressable >
-  );
-
-  const addTodoModal = (<Modal
+  return (<Modal
```

Comment:
This formatting is still off, there should be a new line between `return (` and `<Modal`. Prettier should be automatically fixing these issues?

Reply:
Prettier is working fine. If I define the home file with jsx extension this is formatting as the way you defined above. However, with .js extension it formats like above. I have fixed the formatting as of now.